# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/gimmage/hotfix.patch
# Copyright (C) 2007 - 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- a/src/AppWindow.cpp	2026-10-18 07:11:40.769650529 +0200
+++ b/src/AppWindow.cpp	2026-10-18 07:11:40.772218267 +0200
@@ -30,7 +30,7 @@ extern "C" {
 #include <string.h>
 // we need this to unescape uris!
 #include <curl/curl.h>
-#include <gtk-2.0/gdk/gdkkeysyms.h>
+#include <gdk/gdkkeysyms.h>
 }
 
 
